tests/test_table.py

```diff
@@ -439,6 +439,18 @@ def test_consistent_read(TestModel, TestModel_entries, dynamo_local):
     assert test_model.count == 200
 
 
+def test_delete(TestModel, TestModel_entries, dynamo_local):
```

Comment:
You don't need `TestModel_entries` entries here since you save your own item.

Let's also add `TestModelTwo` here and do the same test on it so that we ensure the delete works for items with only a hash key too.